spec/fixtures/files/datahashes/botgarden/propagation1.json

```diff
@@ -5,14 +5,14 @@
     "propDate": "2020-11-09",
     "propReason": "conservation",
     "propComments": "Be happy",
-    "scarStrat": "cold strat^^boiling water^^",
-    "duration": "25^^10",
-    "durationUnit": "Hours^^Days^^",
```

Comment:
This is a repeating field GROUP (without repeating field SUBGROUP), so we use `;` delimiter

Reply:
Noted.